internal/commands/deprecated_builders.go

```diff
@@ -0,0 +1,13 @@
+package commands
+
+import "github.com/SAP/cloud-mta-build-tool/internal/logs"
+
+// List of deprecated builders
+var deprecatedBuilders = map[string]string{"maven_deprecated": `the "maven_deprecated" builder is deprecated and will be removed after July, 2021.`}
+
+func awareOfDeprecatedBuilder(builder string) {
```

Comment:
Maybe rename to `warnIfDeprecatedBuilder`

Reply:
thought about this, i wanted to be less concrete

Reply:
done